build/assets/configs/99-high-performance-hooks.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "1.0.0",
+  "hook": {
+    "path": "/usr/local/bin/high-performance-hooks.sh",
+    "args": ["high-performance-hooks.sh", "{{.RpsMask}}"]
```

Comment:
probably all occurrences of high-performance should be renamed to be low-latency, I remember @MarSik asked for the same thing for me

Reply:
@MarSik please ack, I don;t care one way or another :)

Reply:
changed, thanks